nmostesting/Config.py

```diff
@@ -129,6 +129,12 @@
 # This will create up to 6 WebSocket servers starting at WEBSOCKET_PORT_BASE up to WEBSOCKET_PORT_BASE + 5
 WEBSOCKET_PORT_BASE = 6000
 
+# For BCP-006-01 testing set EXHAUSTIVE_TESTING to True to run through all
+# TR-08 Capability set/Conformance level/Interop point combinations.
+# Set a RANDOM_SEED to an integer value to make testing deterministic and repeatable.
+EXHAUSTIVE_TESTING = False
+RANDOM_SEED = None
```

Comment:
Do we need to add `EXHAUSTIVE_TESTING`? Please can we reuse `MAX_TEST_ITERATIONS`?
* 0 = EXHAUSTIVE
* N = test that many resource/configs ?

If you always generate resources for all configs, then N becomes sampling from those?
May not be quite as simple as using `sampled_list` - https://github.com/AMWA-TV/nmos-testing/blob/28eeadcd058aa17b6e01ab625b2dd0cdd258585b/nmostesting/NMOSUtils.py#L168
But hopefully similar
And we should fix up `sampled_list` to take account of your new `RANDOM_SEED`?

https://github.com/AMWA-TV/nmos-testing/blob/28eeadcd058aa17b6e01ab625b2dd0cdd258585b/nmostesting/NMOSUtils.py


We'd need to think how we used the seed... you don't want to use the same seed for every use of `sampled_list` I'm guessing... but you want repeatability between runs... how about if you only reran certain test cases so that you might have called `sampled_list` a different number of times? hmmm... at least we'd need to stop using functions in the `random` module directly and use an actual instance of `random.Random` I think...

https://docs.python.org/3/library/random.html#notes-on-reproducibility

Reply:
OK, so it's now using MAX_TEST_ITERATIONS instead of EXHAUSTIVE_TESTING.

The problem with randomness is you're never really going to know...https://xkcd.com/221/